docs/source/examples.rst

```diff
@@ -2,7 +2,14 @@ Examples
 ============
 
 .. toctree::
-   :caption: Table of Contents
 
-   examples/hello_example
-   examples/plots_examples
+   examples/shapes
+   examples/annotations
+   examples/plots
+   examples/text
+   examples/formulas
+   examples/3d
+   examples/camera_settings
+   examples/animations
+   examples/neat_projects
+   examples/advanced_projects
```

Comment:
I would like if this were sorted in a more logical way.

Reply:
I agree.
This can be discussed here:
https://github.com/ManimCommunity/manim/issues/379